locales/hu_HU/LC_MESSAGES/duckduckgo.po

```diff
@@ -4406,7 +4394,7 @@ msgid ""
 "Microsoft's ad network"
 msgstr ""
 "A hirdetéseket a DuckDuckGo adatvédelme keretében tekintheted meg. A "
-"hirdetéskattintásokat a Microsoft hirdetési hálózata kezeli"
+"hirdetésre kattintásokat a Microsoft hirdetési hálózata kezeli"
```

Comment:
@zoli111 At the moment we do not accept translations for strings that relate to legal or contractual obligations, so you should kindly revert this to its previous state. I am sorry this is not explicitly stated in the context.